TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/auto/CompetitionAutonomous.java

```diff
@@ -87,8 +101,15 @@ public void loop() {
                 };
                 break;
             case ALIGN_WITH_CONE:
-                if (align(PixyCam.BLUE, CONE_WIDTH)) {
-                    newState(State.INTAKE_CONE);
+                // if blue team, intake blue cone
+                if (teamState == TeamState.BLUE_LEFT || teamState == TeamState.BLUE_RIGHT) {
```

Comment:
Both the `if` and the `else` are almost the same statement. Consider something like this:

```
int color = TeamColor.BLUE ? PixyCam.BLUE : PixyCam.RED;
if (align(color, CONE_WIDTH)) {
    newState(State.INTAKE_CONE);
}
```